tests/test_psfDetermination.py

```diff
@@ -436,6 +436,44 @@ def testShowPsf(self):
         mos = showPsf(psf, display=testDisplay)
         self.assertTrue(len(mos.images) > 0)
 
+    def testDownsampleBase(self):
+        """Test that the downsampleCandidates function works.
+        """
+        self.setupDeterminer()
+
+        # Try with no downsampling.
+        inputList = list(np.arange(100))
+        candidateList = self.psfDeterminer.downsampleCandidates(inputList)
+        np.testing.assert_array_equal(candidateList, inputList)
+
+        # And with downsampling.
+        inputList = list(np.arange(500))
```

Comment:
This format is convenient for the test, but it's not the same data type as the function actually expects (`lsst.meas.algorithms.PsfCandidate`). I think this is fine, but since many people understand code by looking at how the unit tests work please add a comment stating that this is different from what would typically be supplied as a `candidateList`